docs/DeveloperGuide.md

```diff
@@ -461,20 +461,61 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a student
+### Adding a student
 
-1. Deleting a student while all students are being shown
+1. Adding a student
+   1. Test case: `add n/TestOne p/11111111 e/test1@example.com c/CS2103T`<br>
+   Expected: Student `TestOne` is added to the list. Details of the added student is shown.
+   2. Test case: `add n/TestOne p/11111111`<br>
+   Expected: No student is added. Error details shown.
+   3. Test case: `add n/TestOne e/test1@example.com c/CS2103T`<br>
+   Expected: No student is added. Error details shown.
+   4. Test case: `add n/Test1 p/11111111 e/test1@example.com c/CS2103T`<br>
+      Expected: No student is added. Error details shown.
 
+### Finding a student (by course)
+1. Finding a student (by course)
```

Comment:
Could this be changed to just be the header (with ###), e.g. remove line 477?

Reply:
Sure. Updated as requested.